File: Lista15Questões/go/Atividade1/Main.go
package main

import "fmt"

type Carro struct {
	Marca, Modelo string
	Ano int
}

func (c Carro) exibir_detalhes(){
	fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", c.Marca, c.Modelo, c.Ano)
}

func main(){
	
	carro1 := Carro{
		Marca: "Nissan",
		Modelo: "GTR",
		Ano: 2024,
	}
	
	carro2 := Carro{
		Marca: "Mitsubishi",
		Modelo: "Lancer",
		Ano: 2016,
	}
	
	carro3 := Carro{
		Marca: "Porsche",
		Modelo: "Cayman",
		Ano: 2023,
	}

	carro1.exibir_detalhes()
	carro2.exibir_detalhes()
	carro3.exibir_detalhes()
}

File: Lista15Questões/java/Atividade1/Main.java
package Atividade1;

public class Main {
    
    public static void main(String[] args) {
        
        Carro carro1 = new Carro("Nissan", "Skyline", 1998);
        Carro carro2 = new Carro("Audi", "R8", 2022);
        Carro carro3 = new Carro("Chevrolet", "Camaro", 2023);

        carro1.mostrar_detalhes();
        carro2.mostrar_detalhes();
        carro3.mostrar_detalhes();
    }
}
